<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QDialog" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="name_put">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_put">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>31</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="memo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>메모</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memo_put">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="picture">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>촬영</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Administrator/Pictures/나사렛대학교 로고.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cam">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>CAM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="picture_put">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>포트찾기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
